releasenotes/notes/no_warning_with_reverse_bits-b47cb1e357201593.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    When :func:`~circuit_drawer` was used with `reverse_bits=True` on a
+    circuit without classical bits, warning about `cregbundle` was raised,
+    creating confusion. The warning was removed for this case.
```

Comment:
```suggestion
    Fixed an issue when :func:`~circuit_drawer` was used with ``reverse_bits=True`` on a
    circuit without classical bits that would cause a potentially confusing warning about
    ``cregbundle`` to be emitted.
    Fixed `#8690 <https://github.com/Qiskit/qiskit-terra/issues/8690>`__
```